src/tests/ftest/telemetry/telemetry_pool_metrics.py

```diff
@@ -187,3 +187,24 @@ def test_telemetry_pool_metrics(self):
                 "got=%d, wait_in=[%d, %d]", name, self.dfs_oclass, val, min_val, max_val)
 
         self.log.info("------Test passed------")
+
+    def test_telemetry_pool_metrics_sanity_check(self):
+        """JIRA ID: DAOS-13146
+
+            Create a pool and check whether all the pool metrics listed
+            in the ENGINE_POOL_METRICS are valid.
+        Steps:
+            Create Pool
+            Get all the pool metrics and check for any errors.
+        :avocado: tags=all,daily_regression
+        :avocado: tags=vm
+        :avocado: tags=telemetry
+        :avocado: tags=telemetry_pool_metrics,test_telemetry_pool_metrics_sanity_check
```

Comment:
If you need to push again, please use the class name as a tag
```suggestion
        :avocado: tags=TelemetryPoolMetrics,test_telemetry_pool_metrics_sanity_check
```

Reply:
If I need to push again, I will update the existing test and new test case.

Reply:
Done.